jaas-ljm/index.html

```diff
@@ -3,48 +3,49 @@
-          <input id="tenantInput" name="textinput" type="text" placeholder="vpaas-magic-cookie-..." class="form-control input-md" required="">
-            
+    <form class="form-horizontal" id="form">
+        <fieldset class="m-4">
+          <div class="form-group col-4 my-2 mx-auto">
+            <label class="col-12 control-label" for="regionInput">Region</label>  
+            <select class="form-select" id="regionInput"></select>
           </div>
-        </div>
-        <div class="form-group">
-          <label class="col-md-4 control-label" for="textinput">JWT</label>  
-          <div class="col-md-4">
-          <input id="tokenInput" name="textinput" type="text" placeholder="" class="form-control input-md" required="">
-            
+
+          <div class="form-group col-4 my-2 mx-auto">
+            <label class="col-4 control-label" for="tenantInput"><b>Tenant*</b></label>  
+            <input id="tenantInput" name="tenantInput" type="text" placeholder="vpaas-magic-cookie-..." class="form-control input-md" required>
           </div>
-        </div>
-        <div class="form-group">
-          <label class="col-md-4 control-label" for="textinput">Room</label>  
-          <div class="col-md-4">
-          <input id="roomInput" name="textinput" type="text" placeholder="" class="form-control input-md" required="">
-            
+
+          <div class="form-group col-4 my-2 mx-auto">
+            <label class="col-6 control-label" for="roomInput"><b>Room*</b></label>  
+            <input id="roomInput" name="roomInput" type="text" placeholder="" class="form-control input-md" required>
           </div>
-        </div>
-        <div class="form-group">
-          <label class="col-md-4 control-label" for="singlebutton"></label>
-          <div class="col-md-4">
-            <button id="goButton" type="button" name="singlebutton" class="btn btn-primary">Go</button>
+
+          <div class="form-group col-4 my-2 mx-auto">
+            <label class="col-4 control-label" for="tokenInput">JWT</label>  
+            <input id="tokenInput" name="tokenInput" type="text" placeholder="" class="form-control input-md">
+          </div>
+
+          <div class="form-group col-4 my-2 mx-auto">
+            <label class="col-6 control-label" for="releaseInput">Release version</label>  
+            <div class="col-6 input-group">
+                <input id="releaseInput" name="releaseInput" type="text" placeholder="" class="form-control input-md">
+            </div>
+          </div>
+
+          <div class="form-group col-4 my-3 mx-auto">
+            <div class="col-12 text-center">
+              <button id="goButton" type="button" class="btn btn-primary">Go</button>
+              <button id="hangupButton" type="button" class="btn btn-secondary">Hang up</button>
+            </div>
           </div>
-        </div>
         </fieldset>
     </form>
+    <script id="ljmScript" src="https://8x8.vc/libs/lib-jitsi-meet.min.js"></script>
```

Comment:
Why are you moving this here?

Reply:
Because it will be added at the end of the body when changing the version anyway. Should I change that?

Reply:
Nah, just wanted to understand.